git-branchless-lib/src/testing.rs

```diff
@@ -498,14 +498,15 @@ then you can only run tests in the main `git-branchless` and \
     /// used to set the commit timestamp, which is factored into the commit
     /// hash. The filename is always appended to the message prefix.
     #[instrument]
-    pub fn commit_file_with_contents_and_message(
+    pub fn commit_file_with_contents_and_message_and_file_name(
```

Comment:
I would rather keep the API smaller. Can you revert this change and call `write_file` followed by `commit_file` instead? (Or feel free to use `git.run(&["commit", ...])`; the timestamp should still be stable for tests.)